code/Fluent.php

```diff
@@ -378,7 +378,7 @@ public static function domain_for_locale($locale)
      */
     public static function default_locale($domain = null)
     {
-        if ($domain === true) {
+        if (!Director::is_cli() && $domain === true) {
             $domain = strtolower($_SERVER['HTTP_HOST']);
```

Comment:
as soon as I put `Director::protocolAndHost();` in, I once again receive ` HTTP_HOST not set.  You probably want to define an entry in $_FILE_TO_URL_MAPPING`

I'm quite sure file to url mapping is set correctly and I also double checked it. If it would be wrong, all my custom tasks would also break